core/profiles/standard/standard.install

```diff
@@ -339,6 +318,181 @@ function standard_install() {
+      ),
+      'weight' => -1,
+    ),
+
+    'display' => array(
+      'default' => array(
+        'label' => 'hidden',
+        'type' => 'image',
+        'settings' => array('image_style' => 'large_cropped', 'image_link' => ''),
+        'weight' => 0,
+      ),
+      'teaser' => array(
+        'label' => 'hidden',
+        'type' => 'image',
+        'settings' => array(
+          'image_style' => 'medium',
+          'image_link' => '',
+          'image_float' => '',
+        ),
+        'weight' => -1,
+      ),
+    ),
+  );
+  field_create_instance($instance);
+
+  // Create a default post to display on front page.
+  $post = new Node(
+    array(
+      'title' => st('Your first post!'),
+      'body' => array(
+        LANGUAGE_NONE => array(
+          array(
+            'value' => st('This is your first post! You may edit or delete it.'),
+            'format' => 'filtered_html'
+          ),
+        ),
+      ),
+      'uid' => 1,
+      'status' => 1,
+      'promote' => 1,
+      'type' => 'post',
+    )
+  );
+  $post->save();
+
+  // Create an "About" page node.
+  $page = new Node(
+    array(
+      'title' => st('About'),
+      'body' => array(
+        LANGUAGE_NONE => array(
+          array(
+            'value' => st('This is a page with static content. You may edit or delete it.'),
+            'format' => 'filtered_html',
+          ),
+        ),
+      ),
+      'uid' => 1,
+      'status' => 1,
+      'promote' => 0,
+      'type' => 'page',
+    )
+  );
+  $page->save();
+
+  // Create an "About" link in the main menu.
+  $item = array(
+    'link_path' => 'node/' . $page->nid,
+    'link_title' => 'About',
+    'weight' => 1,
+  );
+  menu_link_save($item);
+
+  // Creating three default Card nodes.
+
+  // Credits for images.
+  // --- @TODO
+  // --- @TODO
+  // --- @TODO
+
+  $field_image_dir = 'public://field/image';
+  file_prepare_directory($field_image_dir, FILE_CREATE_DIRECTORY);
+
+  $cards = array(
+    'card1-organize.png' => array(
+      'title' => st('Organize your content'),
+      'body' => st('<p>Backdrop CMS includes three pre-defined types of content: Pages, Posts, and Cards. Custom content types can also be added.</p><p><a href="https://docs.backdropcms.org/documentation/content-types">Learn more about content types</a></p>'),
+    ),
+    'card2-card.png' => array(
+      'title' => st('What is a Card?'),
+      'body' => st('<p>A Card is a flexible content item. Cards can be put on any page by placing them into a layout as "Existing content" blocks.</p><p><a href="https://docs.backdropcms.org/documentation/blocks">Learn more about blocks</a></p>'),
+    ),
+    'card3-layout.png' => array(
+      'title' => st('The Home page layout'),
+      'body' => st('<p>These Cards on the Home page were added to its layout through a block display from the "Card grid" view.</p><p><a href="https://docs.backdropcms.org/documentation/the-home-page">Learn more about the Home page</a></p>'),
```

Comment:
This information is probably too technical and specific for the first-time persona. They won't know what any of the following are yet: block, display, view." Since the title is about the layout, should we maybe describe that?

Maybe something like... `The home page layout is only active on this page. These cards only appear here`.